tests/selenium_tests.py

```diff
@@ -28,15 +26,15 @@ def setUp(self) -> None:
 
     def test_search_by_valid_data(self):
         # Step_1. Type something in search field on the header of the page
-        self.search_section.search_field().send_keys("Printed dress")
+        self.search_section.fill_search_field(search_phrase="Printed dress")
         # Step_2. Verify that 5 goods are presented in the elastic search dropdown
-        self.assertTrue(self.search_section.search_5th_dropdown_element())
+        self.assertTrue(self.search_section.check_if_dropdown_contains_element(expected_element_count=5))
         # Step_3. Click Search button on the header of the page
         self.search_section.click_search_button()
         # Step_4. Verify that search results block is presented
         self.assertTrue(self.search_page.search_page_title())
```

Comment:
this is not a mistake, just a comment ;)
you can add additional message here that will be printed in case of fail on this assertion
example:
```python
self.assertTrue(self.search_page.search_page_title(), "Missing search result block")
```